core/domain/stats_domain.py

```diff
@@ -95,25 +117,45 @@ def get_multi(cls, exploration_id, rule_data):
 
     @classmethod
     def get(cls, exploration_id, state_name, rule_str):
-        # TODO(sll): Deprecate this method.
+        """Gets domain object corresponding to given rule spec strings and
+        state_name.
+
+        Args:
+            exploration_id: str. The id of the exploration.
+            state_name: str. The current state(card) of the exploration.
+            rule_str: str. Rule spec string which is used to filter the answers
+                based on exploration id and state name.
+
+        Returns:
+            StateRuleAnswerLog object. Get the StateRuleAnswerLog object
+            corresponding to a given exploration_id-state_name and the
+            given rule spec string.
+
+        Todo:
+            ssl: Deprecate this method.
+        """
         return cls.get_multi_by_multi_explorations(
             [(exploration_id, state_name)], [rule_str])[0]
 
     def get_all_top_answers(self):
-        """Returns a list of (answer, count) sorted by count."""
+        """Returns all the answers sorted by count.
 
+        Returns:
+            list. A list of (answer, count) sorted by count.
+        """
         return sorted(
             self.answers.iteritems(), key=operator.itemgetter(1),
             reverse=True)
 
     def get_top_answers(self, num_answers_to_return):
-        """Returns the top `num_answers_to_return` answers.
+        """Returns the top `num_answers_to_return` answers after
+        fetching all top answers from get_all_top_answers().
 
         Args:
             num_answers_to_return: the maximum number of answers to return.
 
         Returns:
-            A list of (answer, count) tuples for the `num_answers_to_return`
-            answers with the highest counts.
+            list. A list of (answer, count) tuples for the
```

Comment:
Ditto -- elaborate on structure. Is the answer a string or a JSON object, for example?